feat(changelog): dedibox-added-ubuntu-2404-lts-noble-numbat-now-availabl 2024-04-29 (#3130)
scaleway/docs-content: changelog/april2024/2024-04-29-dedibox-added-ubuntu-2404-lts-noble-numbat-now-availabl.mdx

---

> title: Ubuntu 24.04 LTS (Noble Numbat) now available!

Reviewer: ```suggestion
title: Ubuntu 24.04 LTS (Noble Numbat) now available on Dedibox servers!
```

status: added
author:
    fullname: ''
    url: 'https://slack.scaleway.com'
date: 2024-04-29
category: bare-metal
product: dedibox
---

> The freshly released Ubuntu 24.04 LTS is now available when you install your Dedibox server 🚀

Reviewer 2: ```suggestion
You can now install the newly launched Ubuntu 24.04 LTS on your Dedibox server 🚀
```
